packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -76,20 +78,19 @@ abstract class CameraPlatform extends PlatformInterface {
     throw UnimplementedError('prepareForVideoRecording() is not implemented.');
   }
 
-  /// Start a video recording and save the file to [path].
+  /// Starts a video recording
```

Comment:
nit: missing a . after the sentence:
```suggestion
  /// Starts a video recording.
```